CHANGES.md

```diff
@@ -4,13 +4,21 @@ wavesurfer.js changelog
 3.4.0 (unreleased)
 ------------------
 
+- Splitted  `regions.js` file in  `region.js` containing the `Region` class, and `region.js` containing the `RegionsPlugin`
+  class, both moved into `src/plugin/regions` directory. In this way creating both classes can be extended to create
+  Custom Plugins.
+
+
+3.3.4 (unreleased)
```

Comment:
No need for a 3.3.4 release first.

Reply:
Maybe 4.0.0 because there are many changes?

Reply:
yeah, you can rename it.